src/care/host_device_map.h

```diff
@@ -66,7 +66,7 @@ namespace care {
    class host_device_map< key_type, mapped_type, RAJA::seq_exec> {
       public:
          // default constructor
-         host_device_map() noexcept = default;
+         host_device_map() noexcept {};
```

Comment:
What is the issue here?  Are you basically getting rid of the GPU constructor?

Reply:
I got this bug during offsite builds, and this was a fix I discussed with Alan Dayton and Jason Burmarck

/p/work1/taller1/JUNE_2024/nativempi/nocheetah/lib/nautilus_gpu/raja/2024.02.0.a/compiler-gcc-8.5.0/cpp-17/blt-0.6.2.d/cmake-3.21.1/camp-2024.02.0.a/cuda-11.6/mpi-openmpi-5.0.0/cub-1.8.0.c/include/RAJA/util/plugins.hpp(28): error: function "lambda [](size_t)->void::<unnamed>(const lambda [](size_t)->void &)" (declared implicitly) cannot be referenced -- it is a deleted function
          detected during:
            instantiation of "auto RAJA::util::trigger_updates_before(T &&)->std::remove_reference<T>::type [with T=lambda [](size_t)->void &]"

Reply:
Yes, exactly

Reply:
This one should be changed back to `= default`

Reply:
Reverted